tests/integration-tests/parsing/test_parsing_service.py

```
@@ -95,37 +95,45 @@ async def test_parse_directory_commit_matches_early_return(self, db_session):
         ), patch(
-            "app.modules.parsing.graph_construction.parsing_service.CodeGraphService",
-            return_value=mock_code_graph,
+            "app.modules.parsing.graph_construction.parsing_service.ParseHelper",
+            return_value=mock_parse_helper,
         ):
             service = ParsingService(
                 db_session, "test-user", raise_library_exceptions=True
             )
             repo_details = ParsingRequest(repo_name="owner/repo")
-            with pytest.raises(ParsingServiceError) as exc_info:
+            with pytest.raises(Exception) as exc_info:
```

Ruff (0.16.0)

[warning] `pytest.raises(Exception)` is too broad, set the `match` parameter or use a more specific exception

(PT011)